roentgen/absorption/material.py

```diff
@@ -0,0 +1,281 @@
+"""
+"""
+
+from __future__ import absolute_import
+
+import numpy as np
+import os
+import astropy.units as u
+from scipy import interpolate
```

Comment:
```suggestion
import os

import astropy.units as u
import numpy as np
from scipy import interpolate

```

`isort` will do that for you.